### PR TITLE
translated the missing strings into Russian

### PR DESCRIPTION
**Pull request by ODK-Bn**

### CODE REVIEW

**Comment by ulteq** [comment]
Did you add yourself to the [AUTHORS.txt](https://github.com/acemod/ACE3/blob/master/AUTHORS.txt)?

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -2703,366 +2703,457 @@
...
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_consumeItem_SurgicalKit_Description">
             <English>Should Surgical kit be removed on usage?</English>
+            <Russian>Нужно ли удалять хирургический набор после использования?</Russian>
             <Polish>Czy zestaw chirurgiczny powinien zniknąć z ekwipunku po jego użyciu?</Polish>
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_useLocation_SurgicalKit_DisplayName">
             <English>Locations Surgical kit (Adv)</English>
+            <Russian>Место использования хирургического набора (усл.)</Russian>
             <Polish>Ogr. zestawu chirurg.</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -2703,366 +2703,457 @@
...
         <Key ID="STR_ACE_AdvancedMedicalSettings_medicSetting_SurgicalKit_Description">
             <English>Who can use the surgical kit?</English>
+            <Russian>Кто может использовать хирургический набор?</Russian>
             <Polish>Kto może skorzystać z zestawu chirurgicznego w celu zszycia ran?</Polish>
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_consumeItem_SurgicalKit_DisplayName">
             <English>Remove Surgical kit (Adv)</English>
+            <Russian>Удалять хирургический набор (усл.)</Russian>
             <Polish>Usuń zest. chir. po użyciu</Polish>
         </Key>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

File: addons/medical/stringtable.xml [hunk 3]
```diff
@@ -2703,366 +2703,457 @@
...
         <Key ID="STR_ACE_AdvancedMedicalSettings_disabled">
             <English>Disabled</English>
+            <Russian>Отключено</Russian>
             <Polish>Wyłączone</Polish>
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_medicSetting_SurgicalKit_DisplayName">
             <English>Allow Surgical kit (Adv)</English>
+            <Russian>Разрешить хирургический набор (усл.)</Russian>
             <Polish>Ust. zestawu chirurg.</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

File: addons/medical/stringtable.xml [hunk 4]
```diff
@@ -2703,366 +2703,457 @@
...
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_Special">
             <English>Doctors only</English>
+            <Russian>Только врачами</Russian>
             <Polish>Tylko doktorzy</Polish>
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_consumeItem_PAK_DisplayName">
             <English>Remove PAK on use</English>
+            <Russian>Удалять аптечки после использования</Russian>
             <Polish>Usuń apteczkę po użyciu</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

File: addons/medical/stringtable.xml [hunk 5]
```diff
@@ -2703,366 +2703,457 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_playersandai">
             <English>Players and AI</English>
+            <Russian>Игроков и ботов</Russian>
             <Polish>Gracze oraz AI</Polish>
         </Key>
         <Key ID="STR_ACE_AdvancedMedicalSettings_enableAdvancedWounds_DisplayName">
             <English>Enable Advanced wounds</English>
+            <Russian>Включить усложненные ранения</Russian>
             <Polish>Akt. zaawansowane rany</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

File: addons/medical/stringtable.xml [hunk 6]
```diff
@@ -2703,366 +2703,457 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_enabled">
             <English>Enabled</English>
+            <Russian>Включено</Russian>
             <Polish>Włączone</Polish>
         </Key>
         <Key ID="STR_ACE_MedicalSettings_preventInstaDeath_DisplayName">
             <English>Prevent instant death</English>
+            <Russian>Отключить мгновенную смерть</Russian>
             <Polish>Wył. natychmiast. śmierć</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: This will not fit the text box inside module (name will be cut)

**Comment by gienkov** [comment]
I pointed out some names that will simply not fit available space in module option name text box. You really should build ACE with your changes and see for yourself what I mean.

**Pull request merged**